### PR TITLE
Intensive_care_uat desadv qty in invoice uom

### PR DESCRIPTION
**Pull request by dm-ps**

### CODE REVIEW

**Review by metas-ts — CHANGES_REQUESTED** [review]
the code LGTM as far as i understood it - besides that one div-by-0 concern.
 
What i'm not sure about from the (test-)code and documented dev-test though
is whether it actually work with >1 LUs with those 3 scenarios that were outlined in the TC.
But if you are optimistic in this regard, let's verify it in the UAT-docu and save the time in the dev-phase.

File: backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/handlingunits/process/WEBUI_M_ReceiptSchedule_ReceiveHUs_UsingConfig.java [hunk 1]
```diff
@@ -53,8 +53,8 @@ public Object getParameterDefaultValue(final IProcessDefaultParameter parameter)
 				return getDefaultLUTUConfiguration().getM_HU_PI_Item_Product_ID();
 			case PARAM_M_LU_HU_PI_ID:
 				return getDefaultLUTUConfiguration().getM_LU_HU_PI_ID();
-			case PARAM_QtyCU:
-				return getDefaultLUTUConfiguration().getQtyCU();
+			case PARAM_QtyCUsPerTU:
```
[the remark this answers is not in the document]
Comment (dm-ps) [reply backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/handlingunits/process/WEBUI_M_ReceiptSchedule_ReceiveHUs_UsingConfig.java:56]: verified on local, added to dev docu

File: backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/pporder/process/WEBUI_PP_Order_Receipt.java [hunk 2]
```diff
@@ -73,7 +73,7 @@ public class WEBUI_PP_Order_Receipt
 	@Param(parameterName = PackingInfoProcessParams.PARAM_M_HU_PI_Item_ID)
 	private I_M_HU_PI_Item p_M_HU_PI_Item;
 
-	@Param(parameterName = PackingInfoProcessParams.PARAM_QtyCU, mandatory = true)
+	@Param(parameterName = PackingInfoProcessParams.PARAM_QtyCUsPerTU, mandatory = true)
```
[the remark this answers is not in the document]
Comment (dm-ps) [reply backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/pporder/process/WEBUI_PP_Order_Receipt.java:76]: verified on local, added to dev docu

File: backend/de.metas.business/src/main/java/de/metas/quantity/StockQtyAndUOMQty.java [hunk 3]
```diff
@@ -202,6 +202,19 @@ public StockQtyAndUOMQty min(@NonNull final StockQtyAndUOMQty other)
...
 
+	@Nullable
+	public BigDecimal getUOMToStockRatio()
+	{
+		return Optional.ofNullable(uomQty)
+				.map(uomQuantity -> {
+					final UOMPrecision uomPrecision = UOMPrecision.ofInt(uomQuantity.getUOM().getStdPrecision());
+
+					return uomQuantity.toBigDecimal().setScale(uomPrecision.toInt(), uomPrecision.getRoundingMode())
+							.divide(stockQty.toBigDecimal(), uomPrecision.getRoundingMode());
```
[the remark this answers is not in the document]
Comment (dm-ps) [reply backend/de.metas.business/src/main/java/de/metas/quantity/StockQtyAndUOMQty.java:219]: in such case, IMHO, we shall return 0, pushed the change

**Comment by dm-ps** [comment]
> the code LGTM as far as i understood it - besides that one div-by-0 concern.
> 
> What i'm not sure about from the (test-)code and documented dev-test though is whether it actually work with >1 LUs with those 3 scenarios that were outlined in the TC. But if you are optimistic in this regard, let's verify it in the UAT-docu and save the time in the dev-phase.

Added scenario `2 LUs` in dev docu for situation when >1LUs

**Review by metas-ts — APPROVED** [review]

**Pull request merged**